fil_fungible_token/src/token/mod.rs

```diff
@@ -17,15 +11,30 @@ use fvm_shared::error::ExitCode;
 use fvm_shared::ActorID;
 use num_traits::Signed;
 use num_traits::Zero;
-use std::ops::Neg;
 use thiserror::Error;
 
+use crate::receiver::types::TokenReceivedParams;
+use crate::runtime::messaging::{Messaging, MessagingError};
+use crate::runtime::messaging::{Result as MessagingResult, RECEIVER_HOOK_METHOD_NUM};
+use crate::token::TokenError::InvalidGranularity;
+
+use self::state::{StateError, TokenState};
+
+mod state;
+mod types;
```

Comment:
My IDE (JetBrains rust plugin) reorganised this. What are you using?

Reply:
I think vscode calls out to rustfmt which is what CI uses as a checker

with [group imports](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#group_imports) set to StdExternalCrate the output is close except that it groups self and crate imports together. I'll just add it to our rusfmt.toml if you are able to configure your plugin to use rustfmt.

Reply:
Let's try it, we can go back to default if it's not great.